### PR TITLE
[webview_flutter] Up version from 1.0.6 to 2.0.4

### PR DESCRIPTION
**Pull request by seungsoo47**
* Migrate to Dart 2.12 and Flutter 2.0
* Update Flutter and Samsung copyright information
* Update webview_flutter_tizen to use platform view interface
* Update example and integration_test
* Update webivew_flutter to 2.0.4

### CODE REVIEW

**Comment by bwikbs** [comment]
How did you work on this work? Very impressive 😱

File: packages/webview_flutter/tizen/src/webview.cc [hunk 1]
```diff
@@ -802,6 +826,7 @@ void WebView::HandleMethodCall(
       std::string js_string = std::get<std::string>(arguments);
       webview_instance_->EvaluateJavaScript(
           js_string, [res = result.release()](std::string value) {
+            value = '"' + value + '"';
```
Comment (bwikbs) [inline packages/webview_flutter/tizen/src/webview.cc:829]: Why? We need this?

Comment (seungsoo47) [reply packages/webview_flutter/tizen/src/webview.cc]: It's my mistake. I fixed it.

**Review by bbrto21 — APPROVED** [review]
Thanks for your work!
LGTM

**Review by bwikbs — APPROVED** [review]

**Pull request merged**